src/WordDaze.Client/WordDaze.Client.csproj

```
@@ -1,15 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

As far as I'm aware Blazor WebAssembly isn't yet supporting .NET Standard 2.1. 
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```